src/main/java/duke/Command.java

```diff
@@ -0,0 +1,54 @@
+package duke;
+
+public enum Command {
+    DONE {
+        public String toString() {
+            return "done";
```

Comment:
Just a small nit but including documentation about how the class is meant to be used could be helpful for other developers (in theory). Sorry for commenting on this again, 2103 required review comments and not general ones.